app/controllers/application_controller.rb

```diff
@@ -49,6 +49,7 @@ def super_admin?
   end
 
   def redirect_user_with_no_organisation
+    return if current_user.nil?
```

Comment:
Could be a line liner? (untested)
```suggestion
    return if current_user.nil? || current_user.is_super_admin? && current_organisation.present?
```

Reply:
Could do, but I'm not sure if it would make the code easier to understand

Reply:
the 2 returns could also be confusing.  No worries, all good.